src/controllers/update/check.your.answers.controller.ts

```diff
@@ -3,15 +3,20 @@ import { NextFunction, Request, Response } from "express";
 import { logger } from "../../utils/logger";
 import { Session } from "@companieshouse/node-session-handler";
 
-import { createOverseasEntity } from "../../service/overseas.entities.service";
+import { ApplicationData } from "../../model";
+import { getApplicationData } from "../../utils/application.data";
+import { isActiveFeature } from "../../utils/feature.flag";
+import { createOverseasEntity, updateOverseasEntity } from "../../service/overseas.entities.service";
+import { OverseasEntityKey, Transactionkey } from "../../model/data.types.model";
 import { closeTransaction, postTransaction } from "../../service/transaction.service";
 import { startPaymentsSession } from "../../service/payment.service";
 
 import {
   CHS_URL,
   UPDATE_BENEFICIAL_OWNER_TYPE_URL,
   UPDATE_AN_OVERSEAS_ENTITY_URL,
-  UPDATE_CHECK_YOUR_ANSWERS_PAGE
+  UPDATE_CHECK_YOUR_ANSWERS_PAGE,
+  FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022
```

Comment:
Only comment would be that this is the same 'save and resume' feature flag that's used for the registration journey and that functionality is hopefully going LIVE soon. So flag would then be set to TRUE in LIVE but potentially FALSE in some of your Kainos envs (dev and test).

You could define a separate, new feature flag for update 'save and resume' to avoid this situation but you may not deem this necessary. Up to you.